cmd/apps.go

```diff
@@ -50,7 +50,7 @@ func (d DeisCmd) AppCreate(id, buildpack, remote string, noRemote bool) error {
 
 	if !noRemote {
 		if err = git.CreateRemote(s.Client.ControllerURL.Host, remote, app.ID); err != nil {
-			if err.Error() == "exit status 128" {
+			if strings.Contains(err.Error(), fmt.Sprintf("fatal: remote %s already exists.", remote)) {
```

Comment:
In this case it isn't possible without larger rewrite. We could map STDERR parsing to errors in `git` package in the future, but I would like to rethink the git package in the future.